src/main/java/codechicken/nei/BookmarkContainerInfo.java

```diff
@@ -0,0 +1,31 @@
+package codechicken.nei;
+
+import java.util.HashMap;
+
+import net.minecraft.client.gui.inventory.GuiChest;
+import net.minecraft.client.gui.inventory.GuiContainer;
+
+import codechicken.nei.api.API;
+import codechicken.nei.api.IBookmarkContainerHandler;
+
+public class BookmarkContainerInfo {
+
+    static final HashMap<Class<? extends GuiContainer>, IBookmarkContainerHandler> handlerMap = new HashMap<>();
```

Comment:
Any reason this is package-private and not completely private? Also: final field names are all upper-case by convention.